pus-note-fr.csl

```diff
@@ -0,0 +1,711 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="never" default-locale="fr-FR">
+
+  <!-- === Metadatas ======================================================= -->
+
+  <info>
+    <title>Presses Universitaires de Strasbourg (note, no abstract, ibid, French)</title>
+    <id>http://www.zotero.org/styles/pus-note-fr</id>
+
+    <link href="http://www.zotero.org/styles/pus-note-fr" rel="self"/>
+    <link href="https://git.unistra.fr/dev-pus/styles-csl/blob/master/README.md#pus-note-ibid-france" rel="documentation" />
+
+    <author>
+      <name>Étienne Nadji</name>
+      <email>etnadji@eml.cc</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>
+      Style for the Presses Universitaires de Strasbourg, 
+      according to 2019 guidelines for authors, 
+      based on "ISO 690 (note, no abstract, French)" by Mathieu Saby, 
+      &lt;http://www.zotero.org/styles/iso690-note-fr&gt;.
+    </summary>
+    <updated>2019-11-19T00:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+
+  <!-- ===================================================================== -->
+
+
+  <!-- === Terms =========================================================== -->
+
+  <!-- Seuls les termes différents || nouveaux sont listés ici -->
+  <!-- Cf. https://github.com/citation-style-language/locales/blob/master/locales-fr-FR.xml -->
+
+  <locale>
+    <terms>
+      <!-- Termes standards redéfinis -->
+      <term name="in">dans</term>
+      <term name="online">en&#160;ligne</term>
+      <term name="accessed">consulté&#160;le</term>
+      <term name="available at">disponible à l'adresse</term>
+      <term name="editor" form="short">dir.</term>
+      <term name="cited">, cité</term>
+      <term name="translator" form="verb-short">traduction de</term>
+      <term name="translator" form="verb">traduction de</term>
+      <term name="no date" form="short"></term>
+      <!-- Termes non-standards -->
+      <term name="opcit">op.&#160;cit.</term>
+      <term name="artcit">art.cité</term>
+      <term name="doiacronym">DOI&#160;:&#160;</term>
```

Comment:
@etnadji, the style won't validate with non-standard CSL terms. You can use prefixes or verbatim text elements instead. E.g. replace

```xml
  <macro name="doi">
    <text term="doiacronym"/>
    <text variable="DOI"/>
  </macro>
```

by

```xml
  <macro name="doi">
    <text prefix="DOI&#160;:&#160;" variable="DOI"/>
  </macro>
```

or

 ```xml
  <macro name="doi">
    <group delimiter="&#160;:&#160;">
      <text value="DOI"/>
      <text variable="DOI"/>
    </group>
  </macro>
```

Reply:
Thanks for this information ! It worked fine when I used it, maybe the CSL formatter was the one Zotero use.